app/mailers/admin_mailer.rb

```diff
@@ -129,6 +129,10 @@ def send_digest_spam(nodes, frequency_digest)
     end
     moderators = User.where(role: %w(moderator admin)).collect(&:email)
     @nodes = nodes
-    mail(to: moderators, subject: @subject)
+    mail(
+      to: "moderators@#{ActionMailer::Base.default_url_options[:host]}",
+      bcc: moderators,
```

Comment:
small change!

Reply:
@ebarry @jywarren Sorry for the late reply. This is really awesome. Is there any other issue with the digest?? and  I will surely work on its UI in few days 
Thanks @jywarren @ebarry :tada: :heart:

Reply:
Awesome. It seems to be working otherwise!